JSDemos/Demos/Charts/ServerSideDataProcessing/description.md

```diff
@@ -1 +1,9 @@
-In&nbsp;many cases, before being handed out to&nbsp;the client, data is&nbsp;processed on&nbsp;the server. The DevExtreme JavaScript UI components support this scenario. In&nbsp;this demo, the data source of&nbsp;the Chart component loads weather data for a&nbsp;selected month from an&nbsp;OData service. Each time the month is&nbsp;changed, the data source sends a&nbsp;new query to&nbsp;the service.
+In many cases, you need to process data on the server before the chart displays it. The Chart component supports this scenario. 
+
+In this demo, the data source of the Chart loads weather data for a selected month from an OData service. Each time you select a different month in the drop-down menu, the data source sends a new query to the service. To implement this functionality, assign a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object to the Chart [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
+
+In the [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/), implement an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/). This demo shows how to [expand](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#expand) used values and how to [postProcess](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#postProcess) them.
```

Comment:
What is `ODataStore` in the first sentence? It looks like a proper noun but an indefinite article stands before it. I would have rather said "implement an OData storage/store".

> This demo shows how to [expand](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#expand) used values and how to [postProcess](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#postProcess) them.

I find the practice of using API names as verbs in a sentence questionable at the very least. What prevents you from using the words "post process" as a link anchor?

Reply:
The latter variant looks more extensive and explanatory. However, I have the two following points:
1. The first sentence is OK but you do not need a comma.
2. You say "use the expand property". How should I use it? Should I set it to true or do something else, perhaps?

Either way, you got rid of API member name usage as full-fledged sentence members, which is commendable.

Reply:
OK, how about this?

> To load multiple collections at once, set the [expand](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#expand) property to an array with the additional collections' titles.

Reply:
We're almost there. The only real objection I still have is this:

> the additional collections' titles.

Namely, the hanging apostrophe. Look. You meant the following, correct?

> the titles of additional collections.

You can convey the same by the following:

> the additional collection titles.

You've made your text simpler and preserved the original semantics.